cluster-commands.go

```diff
@@ -593,3 +593,74 @@ func (adm *AdminClient) SRStatusInfo(ctx context.Context) (info SRStatusInfo, er
 	err = json.NewDecoder(resp.Body).Decode(&info)
 	return info, err
 }
+
+// SiteReplicationRemove - unlinks a site from site replication
+func (adm *AdminClient) SiteReplicationRemove(ctx context.Context, removeReq SRRemoveReq) (st ReplicateRemoveStatus, err error) {
+	rmvBytes, err := json.Marshal(removeReq)
+	if err != nil {
+		return st, nil
+	}
+
+	reqData := requestData{
+		relPath: adminAPIPrefix + "/site-replication/remove",
+		content: rmvBytes,
+	}
+
+	resp, err := adm.executeMethod(ctx, http.MethodPut, reqData)
+	defer closeResponse(resp)
+	if err != nil {
+		return st, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return st, httpRespToErrorResponse(resp)
+	}
+	return ReplicateRemoveStatus{
+		Success: true,
+	}, nil
```

Comment:
`Success: true` is frivolous since we return err as `nil` so whatever is returned here is a success.

Reply:
removed this